### PR TITLE
Consume TRUD data

### PR DESCRIPTION
**Pull request by bjss-colin**
Implemented with the same contract and logic as the existing `OdsService` for backwards compatibility.

The plan, in the future, is to remove the tuple return type and perform some cleanup.

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.ServiceContracts/Organisations/IOdsService.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.ServiceContracts/Organisations/IOdsService.cs]: Is this needed if nothing else in the file has changed?

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json [hunk 2]
```diff
@@ -62,7 +62,8 @@
     "ApiBaseUrl": "https://directory.spineservices.nhs.uk/ORD/2-0-0",
     "BuyerOrganisationRoleIds": [ "RO98", "RO177", "RO213", "RO272" ],
     "GpPracticeRoleId": "RO177",
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json:64]: Not sure if it matters but everything else in this struct is Pascal Case?

**Review by dssherlock — DISMISSED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.ServiceContracts/Organisations/IOdsService.cs (hunk 1 above)
Comment (ghost) [reply src/NHSD.GPIT.BuyingCatalogue.ServiceContracts/Organisations/IOdsService.cs]: Good spot, can't remember what my thinking/planning was around this.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json (hunk 2 above)
Comment (ghost) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json:64]: At some point we need to update configs and how we handle environment variables as there's definitely room for improvement.

Semantically speaking, and although there are no rules set in stone, pascal case is incorrect for JSON. Camel and snake case are the recommended naming conventions for the sake of consistency.

Comment (dssherlock) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json:64]: Yep fair enough-i cant say i have a strong opinion on it!!

**Review by dssherlock — APPROVED** [review]

**Review by imranbashir01 — APPROVED** [review]

**Pull request merged**